tools/gen_istio_image_list.sh

```diff
@@ -139,7 +139,7 @@ do
              echo "Please provide an Istio release."
              display_usage
           else
-             if [[ "$2" =~ [0-9].[0-9].[0-9] ]];
+             if [[ "$2" =~ [0-9]+.[0-9]+.[0-9]+ ]];
```

Comment:
Shouldn't it be `[0-9]+\.[0-9]+\.[0-9]+`?

Reply:
I see，The period character has a special meaning in regular expressions and needs to be escaped using "\"，Thanks!